### PR TITLE
feat: addLocale method

### PR DESCRIPTION
**Pull request by mykeels**
### Proof of concept for #30 

Decoupling means that since a locale consists of the following properties:

- title (string)
- name (array)
- lorem (array)
- address (array)

A user can easily create his/hers without having to modify the core module by 

```js
ngFaker.addLocale('ef', {
  title: 'Efik',
  name: [
    "Efik1", "Efik2"
  ],
  lorem: [],
  address: []
})
```

### CODE REVIEW

**Comment by BolajiOlajide** [comment]
Thanks for contributing @mykeels 
Discovered a bug recently with Circle CI. Will get that fixed soon and revert here. Thanks

File: index.js [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+const {
+  en,
+  yo,
+  ig,
+  ha
+} = require('./lib/locales');
+
 // create new instance of faker class
-var ngfaker = new NgFaker({ locales: require('./lib/locales') });
+var ngfaker = new NgFaker({
```
Comment (BolajiOlajide) [inline index.js]: Just looked through this @mykeels 

I'm thinking, at some point the data sets is going to get larger as we might be providing more data apart from `title`, `name` and `lorem`...

Won't this be a problem at that point. 

Also, does this mean that when one requires the package all locales will be used at the same time.

Comment (BolajiOlajide) [reply index.js]: Hey @mykeels , Looking through this again is there a way we can set the default local to english and only have the user add the locales when they want.

File: index.js [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+  locales: {
+    en
+  }
+});
+
+ngfaker.addLocale('yo', yo);
+ngfaker.addLocale('ig', ig);
+ngfaker.addLocale('ha', ha);
```
Comment (BolajiOlajide) [inline index.js]: I'm guessing after this point where `ngfaker` is exported, it'll automatically have all the locales and I'm thinking what if the user want only one locale for the dummy data?

File: index.js (hunk 1 above)
Comment (mykeels) [reply index.js]: Wow, I'd forgotten about this PR 😄... Can we can look through it together tomorrow?

File: index.js (hunk 2 above)
Comment (mykeels) [reply index.js]: Maybe it should only add the english locale by default, and should `export` the `yo`, `ig`, `ha` objects, so the user can have the option to add them.

Comment (BolajiOlajide) [reply index.js]: I'll find you tomorrow so we can talk this through. Thanks boss!

File: index.js (hunk 1 above)
Comment (BolajiOlajide) [reply index.js]: Yaaaass!! Legend..

**Comment by mykeels** [comment]
Is this ever going to get resolved? 😄

**Comment by BolajiOlajide** [comment]
Hey @mykeels This has got to be the longest PR review in the history of reviews. 

My sincere apologies for delaying this for over a year.

**Pull request merged**